judge/models/contest.py

```diff
@@ -165,17 +165,30 @@ def is_in_contest(self, user):
         return False
 
     def can_see_scoreboard(self, user):
```

Comment:
Maybe `can_see_own_scoreboard`? Having `can_see_scoreboard` and `can_see_full_scoreboard` feels a bit weird.

Reply:
This has been changed in the latest commit.